types/twilio-video/index.d.ts

```diff
@@ -471,21 +473,43 @@ export type AudioTrackPublication = LocalAudioTrackPublication | RemoteAudioTrac
 export interface ConnectOptions {
     abortOnIceServersTimeout?: boolean;
     audio?: boolean | CreateLocalTrackOptions;
+    automaticSubscription?: boolean;
+    bandwidthProfile?: BandwidthProfileOptions;
     dominantSpeaker?: boolean;
+    dscpTagging?: boolean;
+    enableDscp?: boolean;
     iceServers?: RTCIceServer[];
     iceServersTimeout?: number;
     iceTransportPolicy?: RTCIceTransportPolicy;
     insights?: boolean;
-    maxAudioBitRate?: number | null;
-    maxVideoBitRate?: number | null;
+    maxAudioBitrate?: number | null;
+    maxVideoBitrate?: number | null;
     name?: string | null;
     networkQuality?: boolean | NetworkQualityConfiguration;
+    region?: 'au1' | 'br1' | 'ie1' | 'de1' | 'jp1' | 'sg1' | 'us1' | 'us2' | 'gll' | 'us1-ix' | 'us2-ix' | 'ie1-ix' | 'de1-ix' | 'sg1-ix';
```

Comment:
Where `'us1-ix' | 'us2-ix' | 'ie1-ix' | 'de1-ix' | 'sg1-ix'` are described? I couldn't find them in https://www.twilio.com/docs/video/ip-address-whitelisting#signaling-communication which is referred from the api reference.

Reply:
Here:
https://www.twilio.com/blog/2017/12/introducing-gll-for-group-rooms.html

Reply:
Found them here: https://www.twilio.com/docs/voice/voip-sdk/regions
But now I noticed it's `voice`. I made a mistake maybe...

Reply:
I see. Can we exclude `*-ix` from the type as `ConnectOptions` docs only says [this table](https://www.twilio.com/docs/video/ip-address-whitelisting#signaling-communication) is the list of supported signaling regions?

Reply:
Done